basicswap/ui/page_wallet.py

```diff
@@ -216,6 +222,14 @@ def page_wallet(self, url_split, post_string):
                     page_data["wd_type_from_" + cid] = type_from
                 except Exception as e:  # noqa: F841
                     err_messages.append("Missing type")
+            elif coin_id == Coins.FIRO:
+                try:
+                    type_from = form_data[bytes("withdraw_type_from_" + cid, "utf-8")][
+                        0
+                    ].decode("utf-8")
+                    page_data["wd_type_from_" + cid] = type_from
+                except Exception as e:  # noqa: F841
+                    err_messages.append("Missing type")
```

Comment:
Can deduplicate and change
`elif coin_id == Coins.LTC:` to `elif coin_id in (Coins.LTC, Coins.FIRO):`